commands/hosts/delete.go

```diff
@@ -1,19 +1,29 @@
 package hosts
 
 import (
-	"fmt"
-
 	"github.com/urfave/cli"
+	"github.com/vapor-ware/synse-cli/config"
 )
 
 var hostDeleteCommand = cli.Command{
-	Name: "delete",
-	Usage: "delete a Synse Server instance configuration",
-	Action: cmdDelete,
+Name: "delete",
```

Comment:
Just a question, does make fmt need to be run here? It could be done already.

Reply:
yup, my bad for missing it. just pushed a commit that formatted.